test/load-suite.js

```diff
@@ -1,17 +0,0 @@
-const setup = require( 'setup' );
-
-function requireTestFiles( config, path = '' ) {
-	Object.keys( config ).forEach( ( folderName ) => {
-		const folderConfig = config[ folderName ];
-
-		if ( folderName === 'test' && Array.isArray( folderConfig ) ) {
-			folderConfig.forEach( fileName => require( `${path}test/${fileName}` ) );
-		} else {
-			describe( folderName, () => {
```

Comment:
@aduth: I remember that I followed your with how you approached Redux state tests to remove `Makefile` files there. 
Are there any reason to keep tree like structure? I think we can remove it now to simplify our code.